common/buildcraft/transport/BCTransportRecipes.java

```diff
@@ -51,6 +52,8 @@
 
 public class BCTransportRecipes {
     public static void init() {
+        RecipeSorter.register("buildcrafttransport:facade_swap_recipe", FacadeSwapRecipe.class, Category.SHAPELESS, "after:forge:shapedore");
```

Comment:
remind me to remove this again in 1.12 once this is merged (if i forget, might spot it in the merge conflicts this file will have)